### PR TITLE
Tpp private

### PR DESCRIPTION
**Pull request by ZsomborHajzer**
very similar to  Tpp-public already fixed a lot of the mistakes

### CODE REVIEW

File: tpp-private/private.php [hunk 1]
```diff
@@ -18,148 +18,139 @@
...
 <body>
-<div class="container">
+    <div class="container">
 
-    <header>
-        <h1>Placeholder for actual header</h1>
-    </header>
+        <header>
+            <h1>Placeholder for actual header</h1>
+        </header>
```
Comment (AlvesBernardo) [inline tpp-private/private.php:33]: pls no inline style before we forget to take it out

File: tpp-private/private.php [hunk 2]
```diff
@@ -18,148 +18,139 @@
...
-                    <img src="stockphotoholder/addimg.png" alt="" height="200" width="200">
-                    <figcaption>
-                        <p>Add a new image</p>
-                    </figcaption>
-                </label>
-        </div>
-        <!--  <div class="addimg">
+        <div class="photoholder">
+
+            <!--  first img should be this one for talents so they can add more images later on to the project  -->
```
Comment (AlvesBernardo) [inline tpp-private/private.php:53]: class addImg

File: tpp-private/private.php [hunk 3]
```diff
@@ -18,148 +18,139 @@
...
+            <div class="addimg">
+                <form action="#" method="post" enctype="multipart/form-data">
+                    <label>
+                        <input type="file" name="uploadImg" id="" style="display:none">
+                        <img src="stockphotoholder/addimg.png" alt="" height="200" width="200">
```
Comment (AlvesBernardo) [inline tpp-private/private.php:58]: same don't think we should have inline style. let's consult Paul about this as well

File: tpp-private/style.css [hunk 4]
```diff
@@ -76,21 +80,30 @@ header {
...
+
+
 .photoholder {
     display: flex;
     grid-column: 2/6;
     justify-content: space-around;
     flex-wrap: wrap;
 }
 
+
```
Comment (AlvesBernardo) [inline tpp-private/style.css:106]: camelCassing here again documentsTitle would be right way to do it

**Review by AlvesBernardo — CHANGES_REQUESTED** [review]

File: tpp-private/private.php (hunk 1 above)
Comment (ZsomborHajzer) [reply tpp-private/private.php:40]: Sadly it breaks the code so I will try to figure out late

File: tpp-private/private.php (hunk 2 above)
Comment (ZsomborHajzer) [reply tpp-private/private.php:75]: done

File: tpp-private/style.css (hunk 4 above)
Comment (ZsomborHajzer) [reply tpp-private/style.css:106]: done

**Review by AlvesBernardo — APPROVED** [review]

**Pull request closed without merging**